modules/ossm-release-2-6-0.adoc

```diff
@@ -114,12 +114,11 @@ When updating existing instances of the `ServiceMeshControlPlane` resource to {S
 
 {SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortName} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you need to switch to {TempoName} and {OTELName}.
 
-//Gateway API Update for 2.6 OSSM-5854
+//Gateway API Update for 2.6 OSSM-5854 subsequently revised by OSSM-8241
 //Kubernetes Gateway API and {product-title} Gateway API are the same. It is referenced as {product-title} Gateway API in 2.5 and as {product-title} Gateway API here https://docs.openshift.com/container-platform/4.15/nodes/clusters/nodes-cluster-enabling-features.html so to be consistent, it is also referenced as {product-title} Gateway API for 2.6.
 [id="gateway-api-ga-cluster-wide-deployments-ossm-2-6-0_{context}"]
-== {product-title} Gateway API generally available for cluster-wide deployments
-//Jacek. Approved 07/11/2024
-This release introduces the General Availability of {product-title} Gateway API, also known as the Kubernetes Gateway API, which is enabled by default only for cluster-wide deployments.
+== Gateway API use is generally available for OpenShift {SMProductShortName} cluster-wide deployments
+This release introduces the General Availability for using the Kubernetes Gateway API version 1.0.0 with OpenShift {SMProductShortName} 2.6. This API use is limited to OpenShift {SMProductShortName} and the Gateway API custom resource definitions (CRDs) are not supported.
```

Comment:
Should `OpenShift {SMProductShortName}` ("OpenShift Service Mesh") actually be `{SMProductName}` ("Red Hat OpenShift Service Mesh")? (generally "OpenShift" should not appear in plain text)

```suggestion
== Gateway API use is generally available for {SMProductName} cluster-wide deployments
This release introduces the General Availability for using the Kubernetes Gateway API version 1.0.0 with {SMProductName} 2.6. This API use is limited to {SMProductShortName} and the Gateway API custom resource definitions (CRDs) are not supported.
```

Reply:
done